package.json

```diff
@@ -130,6 +130,19 @@
       "type": "string",
       "value": "https://embedly-proxy.services.mozilla.com/v2/extract",
       "hidden": true
+    },
+    {
+      "name": "pocket.endpoint",
+      "title": "Pocket Endpoint",
+      "type": "string",
+      "value": "http://cloudvm.jaredkerim.com/v2/recommendations",
```

Comment:
Do we have a more legit looking dev/stage URL for this? Or at least something w/ HTTPS?
Not that I don't trust @jaredkerim, but...